doc/database_diagnostics.md

```diff
@@ -297,4 +298,56 @@ GET 200 http://localhost:8080/owners?lastName=
             on owner0_.id=pets1_.owner_id 
     where
         owner0_.last_name like ?"], Params:[(%)]
-```
\ No newline at end of file
+```
+
+### Execution Threshold
+
+:wrench: _.properties_ configuration example
+
+You have to configure two properties to set a threshold to SQL executions.
+
+:wrench: .properties configuration
+```properties
+quickperf.database.sql.execution.detected=true
+quickperf.database.sql.execution.threshold=10
+```
```

Comment:
Could you please add a sentence to give the default value for the `quickperf.database.sql.execution.threshold property`? After I will merge.